components/director/internal/systemfetcher/model.go

```diff
@@ -0,0 +1,58 @@
+package systemfetcher
+
+import (
+	"encoding/json"
+
+	"github.com/tidwall/gjson"
+)
+
+var (
+	// Mappings global static configuration which is set after reading the configuration during startup, should only be used for the unmarshaling of system data
+	Mappings []TempMapping
```

Comment:
`TemplateMappings` ?

Reply:
Maybe also add to the godoc what its purpose, because it is a little controversial how we bootstrap it with the global variable 😄